helm/templates/_resources.tpl

```diff
@@ -0,0 +1,34 @@
+{{/*
```

Comment:
Why are you using this file? remove it. It is unnecessary

Reply:
After consulting with @AnguIar and looking threw bitnami implements, we decided to keep this line of method.